texvcjs_readinfile.js

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env node
+var mathStart = {"\\[":"\\]","\\(":"\\)","$":"$","$$":"$$","\\begin{equation}":"\\end{equation}","\\begin{equation*}":"\\end{equation*}","\\begin{align}":"\\end{align}","\\begin{align*}":"\\end{align*}","\\begin{multline}":"\\end{multline}","\\begin{multline*}":"\\end{multline*}"};
```

Comment:
I think the name is not optimal. You have a key value store where the key is the start tag and the value is the end tag.
Also newlines are cheap:-) Putting each element in a new row would help to make it more readable.